experiment/objective.md

```diff
@@ -1,3 +1,3 @@
 1. The objective of this experiment is to understand working of the state digram.
-2. Try to make circuit for following sequence in state digram(000,001,010,100,101,110,000) .
+2. Try to make circuit for the given state diagram in the practice section.
```

Comment:
I don't feel this line has its right place here but since you did not introduce it, we will keep it here